html_app/ui/FacsView.js

```diff
@@ -1145,9 +1152,8 @@ scb.ui.static.FacsView.evaluate_chart = function (state) {
                     // is it over line?
                     console.info("SET TO " + px);
                     var to = px;
-                    to = to > 0 ? to : 0;
-                    to = to > scb.ui.static.FacsView.MAX_GATE  ? scb.ui.static.FacsView.MAX_GATE  : to;
-                    to = to < 0 ? 0 : to;
+                    if(to < 0){ to = 0;}
+                    else if(to > max_x){ to = max_x; }
```

Comment:
nitpick (i.e. safe to ignore): I generally prefer more lines to less for conditionals:
```javascript
if(to < 0) {
    to = 0;
} else if(to > max_x) {
    to = max_x;
}
```
but this is still more clear than the ternary operator